docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
@margaretmoser considered it, but it's not actually a pipe. The output of one is _not_ the input of another. Instead they all share the same input and their outputs are merged together.